dist/game/constants.d.ts

```diff
@@ -181,9 +181,10 @@ declare module "game/constants" {
   export const TOWER_RANGE: number;
   export const TOWER_COOLDOWN: number;
 
-  export const RESOURCE_ENERGY: "energy";
+  export type RESOURCE_ENERGY = "energy";
+  export const RESOURCE_ENERGY: RESOURCE_ENERGY;
 
-  export type ResourceConstant = typeof RESOURCE_ENERGY | typeof RESOURCE_SCORE;
+  export type ResourceConstant = RESOURCE_ENERGY | RESOURCE_SCORE | RESOURCE_SCORE_X | RESOURCE_SCORE_Y | RESOURCE_SCORE_Z;
```

Comment:
We might want to invent a `ARENA_RESOURCE` union type we can import here in game, so it can be specified in the arena module?

Reply:
To be consistent with the other union types, I added `ArenaResourceConstant`.